gallery/visualtests/nxList.js

```diff
@@ -15,6 +15,7 @@ describe('nx-list', function() {
       clickableSelector = '#nx-list-clickable-example .nx-list',
       clickableLinksSelector = '#nx-list-clickable-links-example .nx-list',
       bulletedSelector = '#nx-list-bulleted-example .gallery-example-live',
+      definitionListSelector = '#nx-list-definition-example .nx-list',
```

Comment:
This selector doesn't seem to match anything, and the description list example is already covered

Reply:
> This selector doesn't seem to match anything, and the description list example is already covered

This has been removed.
8e91e31d35537cb6b7c065cfb975e7e62b3da59f